integration/inspektor-gadget/integration_test.go

```diff
@@ -160,7 +160,7 @@ func testMain(m *testing.M) int {
 			patchWebhookConfig = true
 		}
 		initCommands = append(initCommands, DeploySPO(limitReplicas, patchWebhookConfig, bestEffortResourceMgmt))
-		cleanupCommands = append(cleanupCommands, CleanupSPO)
+		cleanupCommands = append(cleanupCommands, CleanupSPO, PurgeSPO)
```

Comment:
Mmmm, what about adding this forcing before trying to delete the SP in the `CleanupSPO` operation? Right before this:

https://github.com/inspektor-gadget/inspektor-gadget/blob/7ba006c31b5cf51ec0ded76570ab17bcaf143813/integration/command.go#L175 

Having this step within the `CleanupSPO` operation would also simplify also the effort to avoid deleting the SPO if we didn't deploy it (https://github.com/inspektor-gadget/inspektor-gadget/pull/1110).

BTW, I think you can patch all the SP (regardless of the namespace) as the `CleanupSPO` operation will try to remove all of them.

What do you think?

Reply:
> Mmmm, what about adding this forcing before trying to delete the SP in the `CleanupSPO` operation? Right before this:
> 
> https://github.com/inspektor-gadget/inspektor-gadget/blob/7ba006c31b5cf51ec0ded76570ab17bcaf143813/integration/command.go#L175
> 
> Having this step within the `CleanupSPO` operation would also simplify also the effort to avoid deleting the SPO if we didn't deploy it (#1110).
> 
Agreed. I was also thinking the same, The main reason we can't run this before `kubectl delete seccompprofile --all --all-namespaces` is spo operator will add the finalizers back so we really need to run both the things in parallel. But I have an idea to combine it, let me update the PR and we see how it is looking?

> BTW, I think you can patch all the SP (regardless of the namespace) as the `CleanupSPO` operation will try to remove all of them.
> 
> What do you think?

The `kubectl patch` doesn't support `--all-namespaces` so I have go over all the namespaces one by one or am I missing something?

Reply:
> The kubectl patch doesn't support --all-namespaces so I have go over all the namespaces one by one or am I missing something?
>
Oops, you are right. Maybe we could get all the ns where the sp are installed using `kubectl get seccompprofile -A`. IIRC, it also shows the ns.

Reply:
Thanks for pointing out. Yes, makes sense to get the namespaces via `kubectl get seccompprofile -A`, directly iterating with `sh` is tricky so let me do it other way around.